distribution/docker/src/docker/Dockerfile

```diff
@@ -257,6 +259,21 @@ RUN sed -i -e 's/ES_DISTRIBUTION_TYPE=tar/ES_DISTRIBUTION_TYPE=docker/' bin/elas
     find . -xdev -perm -4000 -exec chmod ug-s {} + && \\
     find . -type f -exec chmod o+r {} +
 
+<% if (docker_base == "cloud") { %>
+# Preinstall common plugins
+COPY repository-s3-${version}.zip repository-gcs-${version}.zip repository-azure-${version}.zip /tmp/
```

Comment:
This is a question for @mieciu ho's probably on top of this already. Will this work with the existing startup script in cloud ? Once we start using these images, they need to work with already released versions of ECE so while we can and should  make improvements to this script on the cloud side, we have to be backwards compatible too. I'm thinking we might also need a way to tell cloud here that these are already installed.

Reply:
Indeed, that's something that's not going to work for us, at least for `7.x`.

Reply:
Can you advise on what would work?

Reply:
Perhaps `allocator.sh` could be patched to never add or remove these plugins?

Reply:
I think that's the way to go, I pinged @mieciu because I think that can be handled on the cloud side. We add this script as part of the stack-pack so we can make changes to it that will take affect on older allocators too.

Reply:
If it's tricky to modify `allocator.sh` for some reason, we could add logic to `plugin-wrapper.sh` here.